features/hyphens.yml.dist

```diff
@@ -5,12 +5,36 @@ status:
   baseline: low
   baseline_low_date: 2023-09-18
   support:
-    chrome: "55"
+    chrome: "88"
     chrome_android: "55"
-    edge: "79"
+    edge: "88"
```

Comment:
The only weird thing here is that caniuse shows Edge as fully supported only since version 105. It says this about Edge 88 to 104: "Only supported on Android & Mac platforms". I think that might be a bug on caniuse. What do you think?

Reply:
Hmm, it looks like Caniuse is correct - https://github.com/Fyrd/caniuse/pull/5793, but I can't find any mention of the issue in BCD. I think this would be a good candidate for further investigation in https://github.com/web-platform-dx/web-features/issues/1499.